src/components/chatbox/message.module.css

```diff
@@ -77,23 +94,9 @@
 	gap: 16px;
 }
 
-.ratingLike {
-	border: 1px solid var(--token-color-border-success) !important;
-	background: var(--token-color-surface-success) !important;
-	color: var(--token-color-foreground-success) !important;
-}
-.ratingDislike {
-	border: 1px solid var(--token-color-border-critical) !important;
-	background: var(--token-color-surface-critical) !important;
-	color: var(--token-color-foreground-critical) !important;
-}
-
-.feedbackForm {
-	&.feedbackFormVisible {
-		margin-top: 30px;
-	}
-}
-
-.iconError {
-	--icon-color: var(--token-color-foreground-critical);
+.applicationMessage {
+	padding-top: 12px;
+	display: flex;
+	flex-direction: row;
+	gap: 8px;
```

Comment:
Added styles address 
[✓ Global: 429- adjust the language and red icon that displays for this state. ](https://app.asana.com/0/0/1205523765248052/f)